src/homepage/index.js

```diff
@@ -1,7 +1,11 @@
 import React from "react"
 import "./ColumnLayout.css"
 import "./Home.css"
+import { Hero } from "./Hero"
+import { QuickLinks } from "./QuickLinks"
 import { AnnouncementBanner } from "./AnnouncementBanner"
+import { FilerInfo } from "./ForFilers/FilerInfo"
+import { DataInfo } from "./ForDataUsers/DataInfo"
 import { ColumnLayout, Column } from "./ColumnLayout.jsx"
```

Comment:
Lines 9 - 20 has a lot of unused components/code. If we are certain we won't be using a lot of these components can we also remove these components from the codebase?

Reply:
I removed the unused imports as they are in the individual components (<DataInfo /> and <FilerInfo />)